libandroid-navigation-ui/src/test/java/com/mapbox/services/android/navigation/ui/v5/camera/DynamicCameraTest.java

```diff
@@ -145,6 +149,44 @@ public void onInformationFromLocationAndProgress_engineCreatesCorrectBearing() t
     assertEquals(100f, bearing, DELTA);
   }
 
+  @Test
+  public void onInformationFromRoute_engineCreatesOverviewPointList() throws Exception {
+    MapboxMap mapboxMap = mock(MapboxMap.class);
```

Comment:
Minor thing - Could we extract some shared "arrangement" code into `private` methods and used it across the tests? This applies to `MapWaynameTest` as well.